doc/isis/frr-isis-sonic-config-support-hld.md

```diff
@@ -0,0 +1,379 @@
+
+```
+sonic:~$ show isis topology
+Area 1:
+IS-IS paths to level-2 routers that speak IP
+Vertex               Type         Metric Next-Hop             Interface Parent
+sonic1                                                          
+172.20.53.0/31       IP internal  0                           sonic1(4)
+172.20.52.0/31       IP internal  0                           sonic1(4)
+sonic2               TE-IS        10     sonic2               PortChannel0121 sonic1(4)
+10.3.159.80/32       IP TE        10     sonic2               PortChannel0121 sonic2(4)
+10.3.159.81/32       IP TE        10     sonic2               PortChannel0121 sonic2(4)
+......
+```
+
+|Command Description|CLI Command      |
+|:------------------|:-----------------|
+|Show summary of ISIS information |show isis summary |
+
+```
+sonic:~$ show isis summary
+vrf             : default
+Process Id      : 4663
+System Id       : 0000.0000.0000
+Up time         : 00:04:31 ago
+Number of areas : 1
+Area 1:
+  Net: 10.0000.0000.0000.0000.0000.0000.0000.0000.0000.00
+  TX counters per PDU type:
+     L2 IIH: 144
+     L2 LSP: 4
+    L2 CSNP: 29
+   LSP RXMT: 0
+  RX counters per PDU type:
+     L2 IIH: 143
+     L2 LSP: 4
+  Drop counters per PDU type:
+     L2 IIH: 1
+  Advertise high metrics: Disabled
+  Level-1:
+    LSP0 regenerated: 3
+         LSPs purged: 0
+    SPF:
+      minimum interval  : 1
+    IPv4 route computation:
+      last run elapsed  : 00:04:25 ago
+      last run duration : 111 usec
+      run count         : 3
+    IPv6 route computation:
+      last run elapsed  : 00:04:25 ago
+      last run duration : 23 usec
+      run count         : 3
+  Level-2:
+    LSP0 regenerated: 4
+         LSPs purged: 0
+    SPF:
+      minimum interval  : 1
+    IPv4 route computation:
+      last run elapsed  : 00:04:21 ago
+      last run duration : 45 usec
+      run count         : 9
+    IPv6 route computation:
+      last run elapsed  : 00:04:21 ago
+      last run duration : 14 usec
+      run count         : 9
+......
+```
+
+|Command Description|CLI Command      |
+|:------------------|:-----------------|
+|Show ISIS running configuration |show run isis  {--verbose} {--config_db} {--namespace}|
+
+```
+sonic:~$ show run isis
+"""Building configuration...
+Current configuration:
+!
+frr version 8.2.2
+frr defaults traditional
+hostname vlab-01
+log syslog informational
+log facility local4
+no service integrated-vtysh-config
+!
+password zebra
+enable password zebra
+!
+interface PortChannel101
+ ip router isis 1
+ ipv6 router isis 1
+ isis network point-to-point
+exit
+!
+router isis 1
+ is-type level-2-only
+ net 49.0001.1720.1700.0002.00
+ lsp-mtu 1383
+ lsp-timers level-1 gen-interval 30 refresh-interval 900 max-lifetime 1200
+ lsp-timers level-2 gen-interval 30 refresh-interval 305 max-lifetime 900
+ log-adjacency-changes
+exit
+!
+end
+  
+sonic:~$ show run isis  --config_db 
+{ 
+  "ISIS_GLOBAL": { 
```

Comment:
Can we include VRF as the key in the initial schema itself? we can support just default VRF for now.

Reply:
Included support for VRF similarly to how it was supported in the sonic-bgp-global yang model